i18n/locale-en.json

```diff
@@ -0,0 +1,265 @@
+  "I18N_SIGNUP_AGREE_LICENSE_DESCRIPTION": "By checking the box to the left of this text, you acknowledge, agree, and accept to be bound by the <[sitename]> Terms of Use, found <a href=\"/terms\" target=\"_blank\">here</a>.",
+  "I18N_SIGNUP_BUTTON_SUBMIT": "Submit and start contributing",
+  "I18N_SIGNUP_WHY_LICENSE": "Why CC-BY-SA?",
+  "I18N_SIGNUP_SITE_OBJECTIVE": "<[sitename]> exists to foster the creation and continual improvement of a set of high-quality learning resources that are freely available to anyone.",
+  "I18N_SIGNUP_LICENSE_OBJECTIVE":"Using a <[licenselink]> license allows the content of explorations to be freely copied, reused, remixed and redistributed. The main condition is that if someone remixes, transforms or builds upon the material, they must also distribute their work under the same free license.",
+  "I18N_SIGNUP_WAIVER_OBJECTIVE":"The waiver of the attribution (BY) requirement means that, if someone reuses this work, they are not required to attribute the authors. However, all of your individual contributions to explorations will be available on this site in the exploration change log, and people who reuse the exploration are encouraged (but not required) to include a link to this page.",
+  "I18N_SIGNUP_CLOSE_BUTTON": "Close",
+  "I18N_SIGNUP_LOADING": "Loading",
+  "I18N_SIGNUP_FIELD_REQUIRED": "This field is required.",
+  "I18N_SIGNUP_SEND_ME_NEWS": "Send me news and updates about the site",
+  "I18N_SIGNUP_EMAIL_PREFERENCES": "Email Preferences",
+  "I18N_SIGNUP_DO_NOT_SEND_EMAILS": "Do not send these emails",
+  "I18N_SIGNUP_EMAIL_PREFERENCES_EXPLAIN": "You can change this setting at any time from your Preferences page.",
+  "I18N_SIDEBAR_HOME_LINK": "Home",
+  "I18N_SIDEBAR_HOME_ABOUT": "About",
+  "I18N_SIDEBAR_PARTICIPATION_PLAYBOOK": "Participation Playbook",
+  "I18N_SIDEBAR_FORUM": "Forum",
+  "I18N_SIDEBAR_FOLLOW_US": "Follow Us",
+  "I18N_SIDEBAR_ADDITIONAL_LINK_SITE_FEEDBACK": "Site Feedback",
+  "I18N_INTERACTIONS_GRAPH_MOVE": "Move",
+  "I18N_INTERACTIONS_GRAPH_ADD_EDGE": "Add Edge",
+  "I18N_INTERACTIONS_GRAPH_ADD_NODE": "Add Node",
+  "I18N_INTERACTIONS_GRAPH_DELETE": "Delete",
+  "I18N_INTERACTIONS_GRAPH_SUBMIT_BUTTON": "Submit",
+  "I18N_INTERACTIONS_GRAPH_RESET_BUTTON": "Reset",
+  "I18N_INTERACTIONS_GRAPH_ERROR_INVALID": "Invalid graph!",
+  "I18N_INTERACTIONS_GRAPH_RESPONSE_EDGES": "<[edges]> edges",
+  "I18N_INTERACTIONS_GRAPH_RESPONSE_EDGE": "<[edges]> edge",
+  "I18N_INTERACTIONS_GRAPH_RESPONSE_VERTICES": "and <[vertices]> vertices",
+  "I18N_INTERACTIONS_GRAPH_RESPONSE_VERTEX": "and <[vertices]> vertex",
+  "I18N_INTERACTIONS_GRAPH_UPDATE_LABEL": "Update Label",
+  "I18N_INTERACTIONS_GRAPH_UPDATE_WEIGHT": "Update Weight",
+  "I18N_INTERACTIONS_IMAGE_CLICK_SELECT": "[Select an image to display]",
+  "I18N_INTERACTIONS_ITEM_SELECTION_NOT_ENOUGH_PLURAL": "Please select at least <[minChoiceNumber]>",
+  "I18N_INTERACTIONS_ITEM_SELECTION_NOT_ENOUGH": "Please select at least one choice",
```

Comment:
See above. This message should be merged with the message above, and use plural syntax (which should be implemented in the i18n framework).

A quick glance indicates there are at least 50 more occurrences of this pattern.